Ошибка: HTTPConnectionPool(host='69', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027ABC5D50>: Failed to resolve '69' ([Errno 11001] getaddrinfo failed)"))